File: level3/PassingCars.js
function solution(A) {
    var pass = 0, index = 0;
    for (var i=0; i<A.length; i++){
        
        if (A[i] === 0)
            index++;
        
        if (A[i] === 1)
            pass += index;
        
        if(pass > 1000000000)
            return -1;
    }
    return pass
}

File: python_solution/level3/passingCars.py
def solution(A):
    # write your code in Python 2.7
    index = 0
    count = 0
    for el in A:
        if el == 0:
            index += 1
        
        if el == 1:
            count += index
        
        if count > 1000000000:
            return -1
    return count
    pass
